config/test_provider.go

```diff
@@ -0,0 +1 @@
+package config
```

Comment:
Does this mean to have source code?

btw, license is missing. You need to run `make lint`.